content/S2016/S-16-event-square.md

```diff
@@ -0,0 +1,20 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-18 
+Category: Events 
+Tags: academic 
+Slug: square-event-s16 
+Author: Bhargavi Dameracharla 
+Summary: Technical workshop on open source HTTP & HTTP/2 client
+
+Square is holding a technical workshop on their open source HTTP & HTTP/2 client for Android and Java applications. 
+Join us to learn more about Okhttp and get a chance to speak with their engineers! Food will be available. 
+(Including vegetarian, vegan, and gluten-free options)
```

Comment:
Just as a stylistic thing, can you remove the parenthetical and write this as a full sentence? i.e. `Food will be available, including vegetarian, vegan, and gluten-free options.`

You will need another newline after this paragraph.